aicsimageio/readers/default_reader.py

```diff
@@ -434,7 +528,13 @@ def _read_immediate(self) -> xr.DataArray:
             metadata = reader.get_meta_data()
```

Comment:
just randomly noticing this:  `get_meta_data` --> `get_metadata` ???  for another PR...

Reply:
Confusing I know but this is the imageio reader not _our_ reader. From line [408 / 502](https://github.com/AllenCellModeling/aicsimageio/pull/224/files/ab31332a14e085fd350a84dbe063a807099e0676#diff-5cd717e7f683cba43271edb4cac57ba093aae1bbcb3eae60da836cdf0f6a7a16R502) we init an imageio reader.

Reply:
oh sorry. my bad.